src/blocks/skeleton/Skeleton.types.ts

```diff
@@ -0,0 +1,70 @@
+import { CSSProperties, ReactNode } from 'react';
+
+import { BlocksColors, ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
+import { FlattenSimpleInterpolation } from 'styled-components';
+
+export type SkeletonResponsiveProps = {
+  /* Sets display css property */
+  display?: ResponsiveProp<CSSProperties['display']>;
+  /* Sets height css property */
+  height?: ResponsiveProp<string>;
+  /* Sets margin css property */
+  margin?: ResponsiveProp<SpaceType>;
+  /* Sets max-height css property */
+  maxHeight?: ResponsiveProp<string>;
+  /* Sets min-height css property */
+  minHeight?: ResponsiveProp<string>;
+  /* Sets max-width css property */
+  maxWidth?: ResponsiveProp<string>;
+  /* Sets min-width css property */
+  minWidth?: ResponsiveProp<string>;
+  /* Sets padding css property */
+  padding?: ResponsiveProp<SpaceType>;
+  /* Sets width css property */
+  width?: ResponsiveProp<string>;
+  /* Sets border radius css property */
+  borderRadius?: ResponsiveProp<string>;
```

Comment:
Not required. Instead we should have a static border radius defined inside the Skeleton to keep it  consistant throughout the app and a circular prop to make the skeleton completely circular.

Reply:
Currently, the skeleton components being used in the app have multiple values of borderRadius being set. I feel that according to current usage we should keep it as a variable instead of having a fixed value.

Reply:
Cool but then borderRadius should be a non responsive prop.

Reply:
@rohitmalhotra1420, I think we can let it remain as a responsive prop because we might need to change borderRadius according to the device width. Can see a few places where media queries are being used for borderRadius in the application.